### PR TITLE
doc/cephfs: explain the various health messages

### PR DESCRIPTION
**Pull request by jcsp**
Signed-off-by: John Spray <john.spray@redhat.com>

### CODE REVIEW

**Comment by jcsp** [comment]
@gregsfortytwo quick review?

File: doc/cephfs/health-messages.rst [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+Description: The MDS has gone into readonly mode and will return EROFS
+error codes to client operations that attempt to modify any metadata.  The
+MDS will go into readonly mode if it encounters a write error while
+writing to the metadata pool, or if forced to by an administrator using
+the *force_readonly* admin socket command.
+
+Message: *N* slow requests are blocked"
+Code: MDS_HEALTH_SLOW_REQUEST,
+Description: One or more client requests have not been completed promptly,
+indicating that the MDS is either running very slowly, or encountering
```
Comment (gregsfortytwo) [inline doc/cephfs/health-messages.rst]: I'd add here something about the RADOS cluster responding slowly. Eg
> indicating that the MDS is running very slowly, or the RADOS cluster is not acknowledging its journal writes, or there is a bug.

File: doc/cephfs/health-messages.rst [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+can remove (trim) the oldest segments.  If this writeback is happening
+too slowly, or a software bug is preventing trimming, then this health
+message may appear.  The threshold for this message to appear is for the
+number of segments to be double ``mds_log_max_segments``.
+
+Message: "Client *name* failing to respond to capability release"
+Code: MDS_HEALTH_CLIENT_LATE_RELEASE, MDS_HEALTH_CLIENT_LATE_RELEASE_MANY
+Description: CephFS clients are issued *capabilities* by the MDS, which
+are like locks.  Sometimes, for example when another client needs access,
+the MDS will request clients to release their capabilities.  If the client
```
Comment (gregsfortytwo) [inline doc/cephfs/health-messages.rst]: s/request clients to release/request clients release/ ?

**Comment by gregsfortytwo** [comment]
@jcsp, a nit and a quick edit, otherwise LGTM.

**Comment by jcsp** [comment]
Thanks, merging with both those edits made

**Pull request merged**